priv/rel/files/boot

```diff
@@ -135,19 +148,19 @@ generate_vm_args() {
         printf "\n## set via NODE_NAME env variable\n-name $NAME\n" >> $VMARGS_PATH
         NAME_TYPE="-name"
     fi
-    
+
     if [ "$COOKIE_MODE" = "ignore" ]; then
-        printf "Cookie will be used from \"$HOME/.erlang.cookie\".\n"
-        sed -i -e '/-setcookie/s/^/## removed: /' $VMARGS_PATH
-        COOKIE=`cat $HOME/.erlang.cookie`
-        printf "\n## set due to \"COOKIE_MODE=ignore\"\n-setcookie $COOKIE\n" >> $VMARGS_PATH
+        printf "No Cookie handling. (\$HOME/.erlang.cookie might apply).\n"
+        #remove the cookie which is set by default from relx
+        sed -i -e 's/^[ \t]*-setcookie/# disabled by exrm-boot: &/' $VMARGS_PATH
```

Comment:
we will not fix inadequate release-packaging in our wrapper. ignore means ignore.